src/test/java/pages/AboutCompanyPage.java

```diff
@@ -0,0 +1,39 @@
+package pages;
+
+import elements.Text;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import util.WaitUtils;
+
+public class AboutCompanyPage extends BasePage {
+
+    @FindBy(xpath = "//p[2]/strong")
+    protected Text requisites;
```

Comment:
It's goof practice to specify element type in the variable name, ex. requisitesLabel, submitButton and etc. Could you please update variables' names?

Reply:
done